Update figma instructions for external users.mdx (#599)
mi6/ic-design-system: src/content/structured/get-started/figma.mdx

---
path: "/get-started/figma"
 
navPriority: 5
 
date: "2023-09-07"
 
title: "Get the Figma kit"
 
subTitle: "Reusable and flexible Figma components used to design, prototype and hand-off designs for apps and digital products."
 
contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/get-started/figma.mdx"
---
 
import figmaFig1 from "./images/assets2x.png";
import figmaFig2 from "./images/libraries2x.png";
 
## About the Figma UI Kit
 

> We've built a Figma UI Kit to use alongside the guidance and code you'll find on this website. It'll make it easier to prototype and build things quickly.

Reviewer: ```suggestion
We've built a Figma UI Kit to use alongside the guidance and code you'll find on this website. Thus, making it easier to prototype and build projects quickly.
```

Author: Going to leave as is so that we follow our 'use simple words' guidance. :)

https://design.sis.gov.uk/styles/content-style/voice-and-tone#swap-formal-words-for-normal-ones

> All Figma resources have been built with our users in mind, with a strong focus on making sure accessibility is embedded from the start. That way what we can provide an amazing experience, usable for everyone.

Reviewer: ```suggestion
All Figma resources have been built with our users in mind, with a strong focus on making sure accessibility is embedded from the start. That way what we can provide an amazing experience; usable for everyone.
```

Reviewer 2: I'm not super solid on grammar - but as a third option could replace the punctuation with "that is"

Author: Agree.

 
## Accessing the library for internal users
 

> The UI Kit is turned on by default for all projects within our community. Check that you are using the latest version.

Reviewer: ```suggestion
The UI Kit is turned on by default for all projects within our community. Ensure that you are using the latest version.
```

Author: Going to leave as is so that we follow our 'use simple words' guidance. :)

https://design.sis.gov.uk/styles/content-style/voice-and-tone#swap-formal-words-for-normal-ones

 
1. In a new or existing Figma design file, click 'Assets' from the left-hand side panel then click the 'Team library' icon (the book).
 
<DoubleDoDontCaution>
<DoDontCaution
    imageSrc={figmaFig1}
    imageAlt="Image of the left-hand side panel in Figma with the 'Assets' tab open and icon buttons on the right of the panel including the 'Team library' button."
    state="none"
  />
</DoubleDoDontCaution>
 
2. Make sure the Intelligence Community Design System library is turned on.
 
<DoubleDoDontCaution>
<DoDontCaution
    imageSrc={figmaFig2}
    imageAlt="Image of the Libraries menu in Figma with the Intelligence Community Design System library turned on."
    state="none"
  />
</DoubleDoDontCaution>
 
## Not part of the Intelligence Community?
 

> So that everyone can use our resources, we've published our [Figma UI Kit]() to the Figma Community. 

Reviewer: ```suggestion
 We've published our [Figma UI Kit]() to the Figma Community, so that everyone can use our resources.
```

> So that everyone can use our resources, we've published our [Figma UI Kit]() to the Figma Community. 

Author: ```suggestion
We've published our [Figma UI Kit]() to the Figma Community, so that everyone can use our resources.
```

> We've published our [Figma UI Kit]() to the Figma Community, so that everyone can use our resources.

Author: ```suggestion
We've published our [Figma UI Kit](https://www.figma.com/community/file/1283021660431259186) to the Figma Community, so that everyone can use our resources.
```

You can make a copy of it in your local files and use the components in your designs. This is a static only version and you won't automatically receive component updates. Check Figma Community regularly for new updates and versions.
 

> 	1. Navigate to the [UK Intelligence Community UI Kit on Figma Community]().

Author: ```suggestion
	1. Navigate to the [Intelligence Community UI Kit on Figma Community](https://www.figma.com/community/file/1283021660431259186).
```

> 	2. Duplicate the library into your drafts and connect your project files. 

Reviewer: Should a screenshot be added so that it can be more visual?

Author: Good idea.

2. Make sure the Intelligence Community Design System library is turned on.

<DoubleDoDontCaution>
<DoDontCaution
    imageSrc={figmaFig2}
    imageAlt="Image of the Libraries menu in Figma with the Intelligence Community Design System library turned on."
    state="none"
  />
</DoubleDoDontCaution>

## Not part of the Intelligence Community?

We've published our [Figma UI Kit](https://www.figma.com/community/file/1283021660431259186) to the Figma Community, so that everyone can use our resources.

You can make a copy of it in your local files and use the components in your designs. This is a static only version and you won't automatically receive component updates. Check Figma Community regularly for new updates and versions.

    1. Navigate to the [Intelligence Community UI Kit on Figma Community](https://www.figma.com/community/file/1283021660431259186).
    2. Duplicate the library into your drafts and connect your project files.

>     3. Use the [swap libraries](https://help.figma.com/hc/en-us/articles/360039150413-Swap-components-and-instances) function in Figma to quickly swap your components from one version to the next.

Reviewer 3: For some reason, this is showing up in a code block in the build
<img width="829" alt="Screenshot 2023-09-12 at 09 21 05" src="https://github.com/mi6/ic-design-system/assets/113986285/c6efbfb6-b795-4f7c-9f0e-b96b9ff53c8e">

  />
</DoubleDoDontCaution>

## Not part of the Intelligence Community?

> We've published our [Figma UI Kit](https://www.figma.com/community/file/1283021660431259186) to the Figma Community, so that everyone can use our resources.

Reviewer 4: Do we need the comma?

'We've published our Figma UI kit to the Figma community so that everyone can use our resources.

Reviewer 4: Also, push out link please.

> You can make a copy of it in your local files and use the components in your designs. This is a static only version and you won't automatically receive component updates. Check Figma Community regularly for new updates and versions.

Reviewer 4: Static only? 

'This is a static version...'

> 1. Navigate to the [Intelligence Community UI Kit on Figma Community](https://www.figma.com/community/file/1283021660431259186).

Reviewer 4: Push out link please?

2. Duplicate the library into your drafts and connect your project files.

> 3. Use the [swap libraries](https://help.figma.com/hc/en-us/articles/360039150413-Swap-components-and-instances) function in Figma to quickly swap your components from one version to the next.

Reviewer 4: Can this be a push out link please?

  />
</DoubleDoDontCaution>

## Not part of the Intelligence Community?

<p>
  We've published our{" "}
  <ic-link
    target="_blank"
    href="https://www.figma.com/community/file/1283021660431259186"
    rel="noreferer noopener nofollow"
  >
    Figma UI Kit
  </ic-link>{" "}

>   to the Figma Community, so that everyone can use our resources.

Reviewer 4: remove ',' it is uneccessary.

</p>
<p>
  You can make a copy of it in your local files and use the components in your
  designs. This is a static version and you won't automatically receive
  component updates. Check Figma Community regularly for new updates and
  versions.
</p>

<ol>
  <li>
    Navigate to the{" "}
    <ic-link
      target="_blank"
      href="https://www.figma.com/community/file/1283021660431259186"
      rel="noreferer noopener nofollow"
    >
      Intelligence Community UI Kit on Figma Community
    </ic-link>
    .
  </li>
  <li>
    Duplicate the library into your drafts and connect your project files.
  </li>
  <li>
    Use the{" "}
    <ic-link
      target="_blank"
      href="https://help.figma.com/hc/en-us/articles/360039150413-Swap-components-and-instances"
      rel="noreferer noopener nofollow"
    >
      swap libraries
    </ic-link>

>     function in Figma to quickly swap your components from one version to the next.

Reviewer 3: ```suggestion
    {" "}function in Figma to quickly swap your components from one version to the next.
```
A space is needed here

    target="_blank"
    href="https://www.figma.com/community/file/1283021660431259186"
    rel="noreferer noopener nofollow"
  >
    Figma UI Kit
  </ic-link>{" "}
  to the Figma Community so that everyone can use our resources.
</p>
<p>
  You can make a copy of it in your local files and use the components in your
  designs. This is a static version and you won't automatically receive
  component updates. Check Figma Community regularly for new updates and
  versions.
</p>

<ol>
  <li>
    Navigate to the{" "}
    <ic-link
      target="_blank"
      href="https://www.figma.com/community/file/1283021660431259186"
      rel="noreferer noopener nofollow"
    >
      Intelligence Community UI Kit on Figma Community
    </ic-link>
    .
  </li>
  <li>
    Duplicate the library into your drafts and connect your project files.
  </li>
  <li>
    Use the{" "}
    <ic-link
      target="_blank"
      href="https://help.figma.com/hc/en-us/articles/360039150413-Swap-components-and-instances"
      rel="noreferer noopener nofollow"
    >
      swap libraries
    </ic-link>{" "}
    function in Figma to quickly swap your components from one version to the
    next.
  </li>
</ol>
<p>

>   If you want to request a new feature or if you notice any issues, you can{" "}

Reviewer 4: If we are talking commas, then same problem exists here. 

Not sure what the comma is separating here...

Reviewer 4: It's either:

'If you want to request a new feature or if you notice any issues you can raise it on github issues.'

or

'If you want to request a new feature, or if you notice any issues, you can raise it on github issues.'

  <ic-link
    target="_blank"
    href="https://github.com/mi6/ic-design-system/issues"
    rel="noreferer noopener nofollow"
  >
    raise it on GitHub issues
  </ic-link>

>   . For more information read <ic-link href="/community/contribute#contribute-to-the-figma-ui-kit">

Reviewer 3: ```suggestion
  . For more information, read <ic-link href="/community/contribute#contribute-to-the-figma-ui-kit">
```

Reviewer 4: Is this necessary? Are there other elements in this sentence that are comma separted? If just the one sentence, I would imagine no comma is needed.
